package/openwrt/files/etc/init.d/smartdns

```diff
@@ -326,6 +326,9 @@ load_service()
 	config_get log_file "$section" "log_file" ""
 	[ -z "$log_file" ] || conf_append "log-file" "$log_file"
 
+	config_get nftset "$section" "nftset" "0"
```

Comment:
no need to add this options, when the nftset configuration item exists, the feature can be automatically enabled